ci_framework/playbooks/02-infra.yml

```diff
@@ -4,9 +4,9 @@
     step: pre_infra
   ansible.builtin.import_playbook: ./hooks.yml
 
-- name: Infra playbook
-  hosts: "{{ cifmw_target_host | default('localhost') }}"
-  gather_facts: false
+- name: Prepare host virtualization
+  hosts: "{{ cifmw_target_host | default('virthosts', true) | default('localhost') }}"
```

Comment:
I don't really see how that one may hit "localhost" value... ?

Reply:
Gate fails if we remove `localhost`.